build_debian.sh

```diff
@@ -647,5 +647,5 @@ fi
 pushd $FILESYSTEM_ROOT && sudo tar czf $OLDPWD/$FILESYSTEM_DOCKERFS -C ${DOCKERFS_PATH}var/lib/docker .; popd
 
 ## Compress together with /boot, /var/lib/docker and $PLATFORM_DIR as an installer payload zip file
```

Comment:
>Compress [](http://example.com/codeflow?start=3&length=8)

Is anything change from compressed to uncompressed? Did you change the image size changing for all platforms?